Getting_Started.md

```diff
@@ -341,6 +341,17 @@ AgentLabUI allows you to deploy agents to Google Cloud's Vertex AI using the Age
     (If you prefer Yarn: `yarn start`)
 3.  This will open the application in your web browser, usually at `http://localhost:3000`.
 
+**Note:** If you encounter Firestore index errors when using the application:
+1. Open the browser's Developer Tools (F12) and check the Console tab
+2. You'll see error messages with links to create missing Firestore indexes
+3. Click on these links to automatically create the required indexes in your Firebase console
+4. Wait for the indexes to build (this may take a few minutes)
+5. Reload the React application
+6. Repeat this process 2-3 times as you may encounter additional index creation links
+7. The required indexes are also defined in the `firestore.indexes.json` file in your project root
+
+This is normal for the first run as Firestore creates indexes on-demand based on your queries.
```

Comment:
https://firebase.google.com/docs/firestore/query-data/indexing#use_the_firebase_cli

The better right way is to just run `firebase deploy --only firestore` but check that first.